src/ansys_sphinx_theme/__init__.py

```diff
@@ -1,55 +1,116 @@
 """This is the ansys-sphinx-theme module."""
-import os
-from pathlib import Path
+import pathlib
+from typing import Dict
+
+import sphinx
 
 from ansys_sphinx_theme.latex import generate_404  # noqa: F401
 
 __version__ = "0.9.dev0"
 
-# get location of this directory
-_this_path = os.path.dirname(os.path.realpath(__file__))
+
+# Declare the fundamental paths of the theme
+THIS_PATH = pathlib.Path(__file__).parent.resolve()
+THEME_PATH = THIS_PATH / "theme" / "ansys_sphinx_theme"
+STATIC_PATH = THEME_PATH / "static"
+STYLE_PATH = STATIC_PATH / "css"
+CSS_PATH = STYLE_PATH / "ansys_sphinx_theme.css"
+TEMPLATES_PATH = THEME_PATH / "_templates"
 
 # make logo paths available
-pyansys_logo_black = os.path.join(_this_path, "static", "pyansys-logo-black-cropped.png")
-pyansys_logo_white = os.path.join(_this_path, "static", "pyansys-logo-white-cropped.png")
-ansys_favicon = os.path.join(_this_path, "static", "ansys-favicon.png")
-ansys_logo_white = os.path.join(_this_path, "static", "ansys_logo_white.pdf")
-ansys_logo_white_cropped = os.path.join(_this_path, "static", "ansys_logo_white_cropped.pdf")
-watermark = os.path.join(_this_path, "static", "watermark.pdf")
-ansys_logo_black = os.path.join(_this_path, "static", "ansys_logo_black_cropped.jpg")
+ansys_favicon = str((STATIC_PATH / "ansys-favicon.png").absolute())
+ansys_logo_black = str((STATIC_PATH / "ansys_logo_black_cropped.jpg").absolute())
+ansys_logo_white = str((STATIC_PATH / "ansys_logo_white.pdf").absolute())
+ansys_logo_white_cropped = str((STATIC_PATH / "ansys_logo_white_cropped.pdf").absolute())
+html_logo = ansys_logo_black
```

Comment:
```suggestion
```
Do we need this?

Reply:
I don't think so. Our projects are always doing:

```python
from ansys_sphinx_theme import ansys_logo_black, pyansys_logo_black
```

No one is doing:

```python
from ansys_sphinx_theme import html_logo
```

Hence, let me apply your suggestion.